packages/charts/src/charts/metric/metric-chart.component.tsx

```diff
@@ -137,50 +133,45 @@ export const MetricChart: FC<Props> = ({
           </motion.div>
         </AnimatePresence>
         {caption && <Text {...captionSettings.typography}>{caption}</Text>}
-        <AnimatePresence>
-          {tooltipVisible && (
-            <motion.div
-              {...TOOLTIP_MOTION}
-              initial={{
-                opacity: 0,
-                x: tooltipPosition.x,
-                y: tooltipPosition.y,
-              }}
-              animate={{
-                x: tooltipPosition.x,
-                y: tooltipPosition.y,
-                opacity: 1,
-              }}
-              style={{
-                position: 'absolute',
-                pointerEvents: 'none',
-              }}
-            >
-              <Tooltip mode={tooltipSettings.mode} hasArrow={false}>
-                <Text {...tooltipSettings.labels.typography}>
-                  {secondaryValueDescription}
-                </Text>
-              </Tooltip>
-            </motion.div>
-          )}
-        </AnimatePresence>
+        {portalContainer &&
+        portalContainer.current &&
```

Comment:
In my opinion we should inverse the dependency here - by moving `<Portal />` inside `<MetricTooltip />` component to avoid code duplication at chart level.